src/assets/styles/theme.scss

```diff
@@ -29,3 +29,4 @@
 @import './modules/gift-card-template';
 @import './modules/responsive-images';
 @import './modules/hero-banner';
+@import './modules/image-with-text';
```

Comment:
I wonder if we should introduce a `sections` folder instead of `modules`?

Reply:
It makes sense! I moved out this file and created `modules` folder.